### DISCUSSION TITLE
Motion API not working in simplest implementation

### DISCUSSION THREAD

**Discussion by gillarf** [ionic-team/capacitor#4838] [Q&A]
Hi everyone

I'm just trying to explore the Motion API.

I installed capacitor 3  and the Ionic tabs app, and @capacitor/motion, nothing else.

When I run the app on the device, rather than acceleration date I just get the log: Device motion event: {"isTrusted":true}

Does anyone have any idea what I am doing wrong?

`

    import { Component } from '@angular/core';
    import { Motion } from '@capacitor/motion';
    
    @Component({
      selector: 'app-tab1',
      templateUrl: 'tab1.page.html',
      styleUrls: ['tab1.page.scss']
    })
    export class Tab1Page {
    
      constructor() {}
    
      async checkMotion(){
        try {
          await DeviceMotionEvent.requestPermission();
        } catch (e) {
          console.log('Error', e);
          return;
        }
    
        Motion.addListener('accel', (event) => {
          console.log('Device motion event:', event);
        });
    
      }
    
    }
`

**Comment by jcesarmobile** [comment]
Some logs are not properly presented in Xcode/Android Studio, check them in Chrome/Safari remote inspectors
